apps/inheritance_app/inheritance_encrypt_data.c

```diff
@@ -364,7 +364,7 @@ static bool decode_inheritance_plain_data(
     const uint8_t *data,
     uint16_t data_size,
     inheritance_encrypt_data_with_pin_plain_data_structure_t *plain_data) {
-  if (NULL == data || NULL == plain_data || 0 == data_size) {
```

Comment:
`data_size` parameter is the size of the pb encoded bytes of encrypted_data_structure, it cannot be zero, if an empty message is incoming the value of `encrypted_data.data.size` would be 0